pg_app/src/main/java/com/ty/pgboot_app/pg_app/service/RoomsService.java

```diff
@@ -71,4 +72,18 @@ public ResponseEntity<ResponseStructure<Rooms>> deleteRoomsById(int id) {
 		}
 		throw new UnableToDeleteException("no such id found in database to delete");
 	}
+
+	public ResponseEntity<ResponseStructure<List<Rooms>>> getAllRooms() {
+		ResponseStructure<List<Rooms>> responseStructure = new ResponseStructure<List<Rooms>>();
+		ResponseEntity<ResponseStructure<List<Rooms>>> responseEntity = new ResponseEntity<ResponseStructure<List<Rooms>>>(
+				responseStructure, HttpStatus.OK);
+		List<Rooms> optional = roomsDao.getAllRooms();
```

Comment:
put reference varaible as listOfRooms, when next time you push it change it

Reply:
ok